index.js

```diff
@@ -55,10 +56,26 @@ function SRIHashAssets(inputNodes, options) {
 SRIHashAssets.prototype = Object.create(CachingWriter.prototype);
 SRIHashAssets.prototype.constructor = SRIHashAssets;
 
+SRIHashAssets.prototype.getBaseHREF = function getBaseHREF(string) {
+  var baseTag = string.match(BASE_CHECK);
+  if (baseTag && baseTag[0]) {
+    var href = baseTag[0].match(HREF_CHECK);
+    var relativePath = href && href[1];
+
+    if (!relativePath) { return null; }
+
+    return this.inputPaths[0] + relativePath;
```

Comment:
Ya, just confirmed (via [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/base)) that it can be relative or absolute.  

I agree with @jonathanKingston, we can opt out of SRI if `<base>` resolves to a path above the project root.